index.php

```diff
@@ -510,9 +510,19 @@
         // Show heading.
         echo $OUTPUT->heading(get_string('importvideos_importheading', 'block_opencast'));
 
-        // Show explanation.
-        echo html_writer::tag('p', get_string('importvideos_sectionexplanation', 'block_opencast') . '<br />' .
+        // Get import mode from the admin setting.
+        $importmode = get_config('block_opencast', 'importmode');
+        // Provide different explanation based on import mode.
+        if ($importmode == 'acl') {
+            // Show explanation for ACL Change approach.
+            echo html_writer::tag('p', get_string('importvideos_aclsectionexplanation', 'block_opencast') . '<br />' .
```

Comment:
When using the same string (see below) 'importvideos_sectionexplanation', you can move this line before the 'if' so that we have less duplicated code (even if it's only one line).